mpdev/internal/docs/docs.go

```diff
@@ -15,19 +15,51 @@
 package docs
 
 // ReferenceShort contains short help text.
-const ReferenceShort = `placeholder`
+const ReferenceShort = `Overview of mpdev commands`
 
 // ReferenceLong contains expanded help text.
-const ReferenceLong = `placeholder`
+const ReferenceLong = `
+mpdev contains commands to both configure and construct artifacts needed for
+publishing to the Google Cloud Marketplace.
+`
 
 // ReferenceExamples contains examples.
-const ReferenceExamples = `placeholder`
+const ReferenceExamples = `
+  # get a package
+  $ mpdev pkg get https://github.com/GoogleCloudPlatform/kpt.git/examples/wordpress wordpress
+  fetching package /examples/wordpress from \
+    https://github.com/GoogleContainerTools/kpt to wordpress
+
+  # set a value in configuration
+  $ mpdev cfg set image wordpress-image
+	set 1 fields
+
+  # apply mpdev configuration from mpdev.yaml file
+  $ mpdev apply -f mpdev.yaml
+  ...
+  all resources have been created
+`
 
 // ApplyShort contains short help text for apply command.
-const ApplyShort = `placeholder`
+const ApplyShort = `Apply a mpdev configuration to a resource by filename or stdin`
```

Comment:
`Updates or creates artifacts defined by the configurations in filename or stdin`